src/main/webapp/WEB-INF/jsp/include/navBar.jsp

```diff
@@ -147,6 +147,15 @@
         realInterval * 1000
     );
 
+	// hacking OC-13156 Fix border style issues in latest Chrome
+    jQuery( document ).ready(function() {
+        if (navigator.userAgent.toLowerCase().indexOf('chrome') > -1) {
+            jQuery('input[type="text"]').each(function(){
+                jQuery(this).css('background-color', 'revert');
+            });
+        }
+    });
+
```

Comment:
Instead of updating the css using javascript like this, can you update the css directly as mentioned here: https://support.google.com/chrome/thread/48974735?hl=en&msgid=49104946?

Reply:
Tried but it does not remove the border and made specific CSS style for chrome also not working that's why I put it on the script for workaround solution since a lot critic happens maybe this feature will be disabled in the future.